config/odp-linux-generic.conf

```diff
@@ -100,4 +100,15 @@ sched_basic: {
 	burst_size_default = [ 32,  32,  32,  32,  32, 16,  8, 4]
 	burst_size_max     = [255, 255, 255, 255, 255, 16, 16, 8]
 
+	# Automatically updated schedule groups
+	#
+	# API specification defines that ODP_SCHED_GROUP_ALL,
+	# _WORKER and _CONTROL are updated automatically. These options can be
+	# used to disable these group when not used. Set value to 0 to disable
+	# a group. Performance may improve when unused groups are disabled.
```

Comment:
Threads are created as worker or control threads and queues are created in a specific scheduler group (default = `ODP_SCHED_GROUP_ALL`) so I'm not sure I understand the use cases envisioned here. What happens if `all = 0` is specified and `odp_queue_create()` is called to create a `SCHED` queue without an explicit scheduling group, so that by default it's assigned to group `ODP_SCHED_GROUP_ALL`?

Omitting the explicit worker and control groups might make sense if queues are either going to be in the default `all` group or explicitly named groups, but being able to disable the `all` group seems error-prone and it's not obvious under what conditions it would be useful.

Reply:
If user/application creates its own groups and places all queues in to those groups. "All" groups is not used or needed. User needs to be careful when changing configuration from the default.

Reply:
Unless the application has complete control over all queue creation (which it does not, since ODP creates queues internally) it seems that allowing the `all` group to be removed is likely to cause problems. Is the performance gain so significant that this is worth it?